server/server.go

```diff
@@ -99,6 +99,10 @@ func (s *Server) handleParse(ctx *gin.Context) {
 		Content(req.Content).
 		Mode(mode).
 		UAST()
+	if bblfsh.ErrSyntax.Is(err) || bblfsh.ErrDriverFailure.Is(err) {
+		ctx.JSON(http.StatusBadRequest, jsonError("error parsing UAST: %s", err))
```

Comment:
Is the meaning of `ErrDriverFailure` a clear `StatusBadRequest`, as in the user provided an input that made the driver fail?
To me it sounds like it should maybe be a `StatusInternalServerError`

Reply:
and here is an issue for LA: https://github.com/bblfsh/python-driver/issues/169

Reply:
great, thanks